android/capacitor/src/main/java/com/getcapacitor/BridgeWebChromeClient.java

```diff
@@ -39,13 +39,12 @@
  */
 public class BridgeWebChromeClient extends WebChromeClient {
 
-    static final int FILE_CHOOSER = PluginRequestCodes.FILE_CHOOSER;
-    static final int FILE_CHOOSER_IMAGE_CAPTURE = PluginRequestCodes.FILE_CHOOSER_IMAGE_CAPTURE;
-    static final int FILE_CHOOSER_VIDEO_CAPTURE = PluginRequestCodes.FILE_CHOOSER_VIDEO_CAPTURE;
-    static final int FILE_CHOOSER_CAMERA_PERMISSION = PluginRequestCodes.FILE_CHOOSER_CAMERA_PERMISSION;
-    static final int GET_USER_MEDIA_PERMISSIONS = PluginRequestCodes.GET_USER_MEDIA_PERMISSIONS;
-    static final int GEOLOCATION_REQUEST_PERMISSIONS = PluginRequestCodes.GEOLOCATION_REQUEST_PERMISSIONS;
-    static final String[] geoPermissions = { Manifest.permission.ACCESS_COARSE_LOCATION, Manifest.permission.ACCESS_FINE_LOCATION };
+    static final int FILE_CHOOSER = 9007;
+    static final int FILE_CHOOSER_IMAGE_CAPTURE = 9008;
+    static final int FILE_CHOOSER_VIDEO_CAPTURE = 9009;
+    static final int FILE_CHOOSER_CAMERA_PERMISSION = 9010;
+    static final int GET_USER_MEDIA_PERMISSIONS = 9011;
+    static final int GEOLOCATION_REQUEST_PERMISSIONS = 9004;
```

Comment:
```suggestion
    static final int FILE_CHOOSER = 9001;
    static final int FILE_CHOOSER_IMAGE_CAPTURE = 9002;
    static final int FILE_CHOOSER_VIDEO_CAPTURE = 9003;
    static final int FILE_CHOOSER_CAMERA_PERMISSION = 9004;
    static final int GET_USER_MEDIA_PERMISSIONS = 9005;
    static final int GEOLOCATION_REQUEST_PERMISSIONS = 9006;
```

better put codes in order for making easier to spot which ones are available in case we need to add more

Reply:
Will these not conflict with the uses that already used the codes in that range? https://github.com/ionic-team/capacitor/pull/3621/files#diff-120f0e6b31193b391266aa177c18c6825b978c72c444be297115b31a061bb3a1

Reply:
In the plugins that used those codes we will need to use different codes from this ones, but plugins need to be updated anyway since this PR removes the file with the constants.
We just need to make sure the codes in the plugins are unique.

Reply:
I think this is good, but let's do it after we have fixed all the plugins to work with the new permissions. Then we can do all the code changes across all plugins in a single commit and centralize it in one place so we don't miss any.

Reply:
Then would be better to revert the removal of PluginRequestCodes.java and remove it once all plugins are migrated

Reply:
Makes sense, reverted for now